purchase_request_procurement_operating_unit/model/procurement.py

```diff
@@ -0,0 +1,47 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016-17 Eficent Business and IT Consulting Services S.L.
+#   (http://www.eficent.com)
+# Copyright 2016-17 Serpent Consulting Services Pvt. Ltd.
+#   (<http://www.serpentcs.com>)
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl.html).
+
+from odoo import models, api, _
+from odoo.exceptions import ValidationError
+
+
+class Procurement(models.Model):
+    _inherit = 'procurement.order'
+
+    @api.model
+    def _prepare_purchase_request(self, procurement):
```

Comment:
This method has changed. Now in v10 is @api.multi, thus, the procurement is passed in self, not as argument.

Reply:
I see, let me do the needful!